src/main/java/io/jenkins/plugins/ml/IPythonInterpreterManager.java

```diff
@@ -57,6 +57,7 @@
     IPythonInterpreterManager(IPythonUserConfig userConfig) {
         this.userConfig = userConfig;
         mockInterpreterGroup.put("session_" + sessionId, new ArrayList<Interpreter>());
+        LOGGER.setLevel(Level.OFF);
```

Comment:
This is the only blocker for me here @Loghijiaha , as that could disable the loggers and hide a legit issue with Zeppelin. All the other changes are good to go IMO.